en/services/camera.md

```diff
@@ -1,42 +1,112 @@
+  Otherwise, it can be hosted by any regular, reachable server. 
 
-### Video Streaming
+The `CAMERA_INFORMATION.cam_definition_version` field should provide a version for the definition file, allowing the GCS to cache it. 
+Once downloaded, it would only be requested again if the version number changes.
+
+If a vehicle has more than one camera, each camera will have a different component ID and send its own heartbeat. 
+The GCS should create multiple instances of a camera controller based on the component ID of each camera. 
+All commands are sent to a specific camera by addressing the command to a specific component ID.
+
+
+## Basic Camera Operations
 
-If a camera is capable of streaming video, it must set the [CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM](../messages/common.md#CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM) flag within the [CAMERA_INFORMATION](../messages/common.md#CAMERA_INFORMATION) message `flags` field to inform the GCS. When the GCS receives the [CAMERA_INFORMATION](../messages/common.md#CAMERA_INFORMATION) message and it detects the [CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM](../messages/common.md#CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM) flag, it will then send the [MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION](../messages/common.md#MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION) message to the camera requesting the video streaming configuration. In response, the camera returns a [VIDEO_STREAM_INFORMATION](../messages/common.md#VIDEO_STREAM_INFORMATION) message for each stream it supports.
+The [CAMERA_INFORMATION.flags](../messages/common.md#CAMERA_INFORMATION) provides information about camera capabilities.
+It contains a bitmap of [CAMERA_CAP_FLAGS](../messages/common.md#CAMERA_CAP_FLAGS) values that tell the GCS if the camera supports still image capture, video capture, or video streaming, and if it needs to be in a certain mode for capture, etc.
 
-> **Note** If your camera only provides video streaming and nothing else (no camera features), the [CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM](../messages/common.md#CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM) flag is the only flag you need to set. The GCS will then provide video streaming support and skip camera control.
 
 ### Camera Modes
 
-Some cameras must be in a certain mode for still and/or video capture. The [CAMERA_INFORMATION](../messages/common.md#CAMERA_INFORMATION) message `flags` field uses the [CAMERA_CAP_FLAGS_HAS_MODES](../messages/common.md#CAMERA_CAP_FLAGS_HAS_MODES) bit set to true to inform the GCS that it needs to make sure the camera is in the proper mode prior to sending a start capture (image or video) command. In addition, some cameras can capture images in any mode but with different resolutions. For example, a 20 megapixel camera would take a full resolution image when set to `CAMERA_MODE_IMAGE` but only at the current video resolution if it is set to `CAMERA_MODE_VIDEO`.
+Some cameras must be in a certain mode for still and/or video capture.
```

Comment:
OK, I also updated this section to add sequence diagrams. You can see just these changes in https://github.com/mavlink/mavlink-devguide/pull/163/commits/f6207b5197bc72c7a5135015e22f5d5151123555

Reply:
And you can see it rendered with sequences here: https://hamishwillee.gitbooks.io/ham_mavdevguide/content/v/cameraupdates/en/services/camera.html#camera-modes